create-regolith-term-profile

```diff
@@ -51,13 +51,16 @@ apply() {
     scrollbar="$(xrescat gnome.terminal.scrollbar never)" \
     useTransparentBg="$(xrescat gnome.terminal.use-transparent-background false)" \
     audibleBell="$(xrescat gnome.terminal.audible-bell false)" \
-    transparentBgPercent="$(xrescat gnome.terminal.background-transparency-percent 4)"
+    transparentBgPercent="$(xrescat gnome.terminal.background-transparency-percent 4)" \
```

Comment:
Does this mean the default transparency will be set to 4%?

Reply:
I believe so, but I only escaped the new line here (\) the 4 is what was already being set